### ISSUE TITLE
Failed to listen on [::1]:50001 with error: listen tcp [::1]:50001. what dose 50001 port and 3500  port do?

### ISSUE THREAD

**Issue by adxpcc** [dapr/dapr#4441]
dapr 1.6

Hi  I followd  official documentation :
https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-overview/
deployd  in my k8s.

In my case  the pods is ok
[root@master dapr-test]# kubectl  get pod -ndotnet
NAME                             READY   STATUS    RESTARTS   AGE
backend-http-874d7f6cf-rlqng     2/2     Running   0          13m
frontend-http-6b58764856-gcfpp   2/2     Running   0          13m

Service Invoke is Ok. State is Ok.

but I found some warning message in dapr sidecar:

`[root@master dapr-test]# kubectl logs frontend-http-6b58764856-gcfpp daprd -ndotnet | grep error
time="2022-03-29T02:11:53.446571144Z" level=warning msg="Failed to listen on [::1]:50001 with error: listen tcp [::1]:50001: bind: cannot assign requested address" app_id=frontend-http instance=frontend-http-6b58764856-gcfpp scope=dapr.runtime.grpc.api type=log ver=1.6.0
time="2022-03-29T02:11:53.446920181Z" level=warning msg="Failed to listen on [::1]:3500 with error: listen tcp [::1]:3500: bind: cannot assign requested address" app_id=frontend-http instance=frontend-http-6b58764856-gcfpp scope=dapr.runtime.http type=log ver=1.6.0`

so what dose the port 50001 and 3500 do?  What the impact this will have?

my deploy yaml
apiVersion: apps/v1
kind: Deployment
metadata:
 name: frontend-http
 namespace: dotnet
spec:
 replicas: 1
 selector:
    matchLabels:
      app: frontend-http
 template:
  metadata:
   annotations:
        dapr.io/enabled: "true"
        dapr.io/app-id: "frontend-http"
        dapr.io/app-port: "80"
        dapr.io/config: "dapr-config"
   labels:
    app: frontend-http
  spec:
   containers:
    - name: frontend-http
      image: a121984376/frontend-http:v4
      imagePullPolicy: IfNotPresent
      ports:
       - containerPort: 80
      livenessProbe:
       httpGet:
        path: /frontend/metrics
        port: 80
       initialDelaySeconds: 10
       periodSeconds: 3
      readinessProbe:
       httpGet:
        path: /frontend/metrics
        port: 80
       initialDelaySeconds: 10
       periodSeconds: 3
---
apiVersion: v1
kind: Service
metadata:
 name: svc-frontend-http
 namespace: dotnet
spec:
 type: NodePort
 selector:
  app: frontend-http
 ports:
  - protocol: TCP
    port: 80
    nodePort: 30066
    targetPort: 80

**Comment by yaron2** [comment]
It's ok to see this, it just means Dapr failed to bind the ports on IPv6. You can ignore these warnings.

**Comment by a121984376** [comment]
> It's ok to see this, it just means Dapr failed to bind the ports on IPv6. You can ignore these warnings.

Thinks

**Comment by kayvonkhosrowpour** [comment]
@yaron2 Is there a way to configure the log level of this from warning to something else? We ingest these logs into our monitoring platform and warning set off alerts.

**Comment by kayvonkhosrowpour** [comment]
Following up to this. @yaron2 Is there a way to change the log level of this? Why do we receive warnings?

**Comment by yaron2** [comment]
> Following up to this. @yaron2 Is there a way to change the log level of this? Why do we receive warnings?

You're right that this probably should become debug logs levels. If you want to submit a PR to change this, that would be great. No worries if not, just let me know and we'll get someone on it.

**Comment by kayvonkhosrowpour** [comment]
Thanks for the quick completion! :)
